javalib/src/main/scala/java/lang/IEEE754Helpers.scala

```diff
@@ -31,21 +33,23 @@ private[java] object IEEE754Helpers {
   private def bytesToCString(bytes: Array[scala.Byte], n: Int)(implicit
       z: Zone
   ): CString = {
-    val cStr = z.alloc((n + 1).toUInt) // z.alloc() does not clear bytes.
+    val cStr = z.alloc((n + 1).toUSize) // z.alloc() does not clear bytes.
 
-    var c = 0
-    while (c < n) {
-      !(cStr + c) = bytes(c)
-      c += 1
-    }
+    // memcpy bytes from Array to CString
+    val bytesPtr = bytes.asInstanceOf[ByteArray].at(0)
```

Comment:
This is a candidate for Arman's PR #2888. I think that has merged for 0.4.n but
not yet 0.5.0.

Reply:
BTW, I'm wondering if there is any performance difference between 
`!(cStr + n) = 0.toByte`
and
`cStr(n) =  0.toByte`

Reply:
I would be astonished if the two did not have exactly the same code, after they
have been through SN and LLVM.

You can tell from the first form that a C developer wrote it.  I  just do not remember
if the second form was available when that code was written. Ptr/Array conversions,
or my understanding of them in SN, have gotten better over time.

Now that the second form is widespread & idomatic, I think it is clearer to read and
comprehend 'on-sight'.  Collegial review _does_ improve things.

Reply:
Agree.

I did the change in the last commit.